rpc/src/main/protobuf/master.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package kr.ac.postech.paranode.rpc;
+
+service Master {
+    
+    rpc RegisterWorkerDirectory (RegisterRequest) returns (RegisterRelpy) {}
```

Comment:
chore: `RegisterRelpy` should fixed to `RegisterReply`

Reply:
https://github.com/betarixm/434project/pull/59/commits/f7b490679dd9d46949c4e52e45f3946ccaf3f8c1